app/javascript/packs/search.tsx

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import LocationList from '../src/components/LocationList/LocationList';
+import { LocationList } from '../src/components/LocationList/LocationList';
 
 const elements = document.getElementsByClassName('search');
 
 // TODO do we need an event listiner? DOMCOntentLoaded or Turbolinks related?
 Array.from(elements).forEach((element) => {
-  const options = JSON.parse(element.getAttribute('data-options'));
```

Comment:
the `data-options` was left in there so we could pass in variables from the ERB render, ie from rails, we probably should delete it as you have done until we actually need it. I suppose I had it in there as a reference when I do need it :) but all good let's delete until we need it

Reply:
in fact we do use this for our Google API key from `app/views/forecast/index.html.erb` so do the maps work on your version with the key?

Reply:
I deleted it because it's not being used and ts doesn't like having stuff around that's not being used...everything was working for me, so it doesn't look like we need it